source/gfoidl.Stochastics/Statistics/Sample.Delta.cs

```diff
@@ -159,5 +152,47 @@ void Core(double* arr, int offset, Vector<double> avgVec, ref Vector<double> del
                 deltaVec          += Vector.Abs(vec - avgVec);
             }
         }
+        //---------------------------------------------------------------------
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private unsafe void CalculateDeltaImplCore4(double* arr, int offset, Vector<double> avgVec, ref Vector<double> deltaVec0, ref Vector<double> deltaVec1, ref Vector<double> deltaVec2, ref Vector<double> deltaVec3, double* end)
+        {
+#if DEBUG_ASSERT
+            // arr is included -> -1
+            Debug.Assert(arr + offset + 4 * Vector<double>.Count - 1 < end);
+#endif
+            // Vector can be read aligned instead of unaligned, because arr was aligned in the sequential pass.
+            Vector<double> vec0 = VectorHelper.GetVector(arr + offset + 0 * Vector<double>.Count);
+            Vector<double> vec1 = VectorHelper.GetVector(arr + offset + 1 * Vector<double>.Count);
+            Vector<double> vec2 = VectorHelper.GetVector(arr + offset + 2 * Vector<double>.Count);
+            Vector<double> vec3 = VectorHelper.GetVector(arr + offset + 3 * Vector<double>.Count);
+
+            vec0 -= avgVec;
```

Comment:
Try if `minusAvgVec` is faster, so that
```c#
vec0 += minusAvgVec
```

Reply:
According to the intel intrinsics guid `vsubpd` and `vaddpd` have the same latency and same throughput. Also I can't measure any difference.

So, for clearness, I'll leave it as is.